packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -47,6 +50,26 @@ export function SquadHeaderBar({
     key: TutorialKey.CopySquadLink,
   });
 
+  const { steps } = useSquadChecklistSteps({ squad });
+
+  const checklistTooltipText = useMemo(() => {
+    const completedStepsCount = steps.filter(
```

Comment:
I have another suggestion here.
Maybe you can abstract the map inside the hook, so it's outside the memo.
Then keep this logic inside the hook (like `completedSteps`)
raising it because down the line you might want it somewhere else then all logic is handled in the hook.

You could also add `totalSteps` for instance

Let me know what you think.

Reply:
We can move this to `useSquadChecklistSteps` sure 👍

Reply:
Streamlined usage even more.